app/src/main/java/ch/epfl/qedit/backend/database/MockDBService.java

```diff
@@ -136,13 +116,88 @@ public void run() {
                                 } catch (InterruptedException e) {
                                     e.printStackTrace();
                                 }
+                                Response<List<Question>> response;
+                                if (!db.containsKey(quizID))
+                                    response = Response.error(WRONG_DOCUMENT);
+                                else {
+                                    List<Question> questions =
```

Comment:
I will try to search on internet if it is possible to prevent `spotless` to create that mess in our code.